config.txt

```diff
@@ -26,18 +26,18 @@ framebuffer_height=720
 display_rotate=0
 lcd_rotate=0
 
-# uncomment if hdmi display is not detected and composite is being output
+# uncomment if HDMI display is not detected and composite is being outputted
 #hdmi_force_hotplug=1
 
 # uncomment to force a specific HDMI mode (this will force VGA)
 #hdmi_group=1
 #hdmi_mode=1
 
-# uncomment to force a HDMI mode rather than DVI. This can make audio work in
+# uncomment to force an HDMI mode rather than DVI. This can make audio work in
 # DMT (computer monitor) modes
 #hdmi_drive=2
 
-# uncomment to increase signal to HDMI, if you have interference, blanking, or
+# uncomment to increase the signal to HDMI if you have interference, blanking, or
```

Comment:
For our defence, the above is from default pre-installed `config.txt` on Raspbian 😅. However, somehow the cases were lost: https://elinux.org/R-Pi_configuration_file#An_example_configuration_file